portfolio/src/main/java/com/google/sps/classes/AllowedLanguageCodes.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.classes;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class AllowedLanguageCodes {
+  private static final Language[] translationLangCodes = { new Language("English", "en", "en-US", 0),
+      new Language("Romanian", "ro", "ro-RO", 1), new Language("German", "de", "de-DE", 2),
+      new Language("French", "fr", "fr-FR", 3) };
+
+  public static List<Language> getTranslationLanguageList() {
+    List<Language> lst = new ArrayList<>();
+    for (Language it : translationLangCodes)
+      lst.add(it);
+    return lst;
```

Comment:
Better leverage the JDK: `return Collections.unmodifiableList(Arrays.asList(translationLangCodes));`